pkg/providers/oracle/migrate_tables.go

```diff
@@ -0,0 +1,37 @@
+package oracle
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/banzaicloud/pipeline/pkg/providers/oracle/model"
+	objectstoreModel "github.com/banzaicloud/pipeline/pkg/providers/oracle/model/objectstore"
```

Comment:
`objectstore` looks enough here

Reply:
I used `objectstoreModel` just to be clear on which package is that. In the current structure the `objectstore` model should be in the `oracle/model` package anyway, but putting it there would cause cyclic imports right now.